docs/source/setup/manual_way.rst

```diff
@@ -1,55 +1,60 @@
 Manual Way
 ************
+### Quick install
+Its simplicity makes SQLite extremely useful for development. It speeds up the setup of the development environment.
+To start Papermerge for test or development from zero - I need to clone repo, create virtualenv, run migrations and Done!
 
-If you follow this document and still have troubles, please open an
-`issue on GitHub: <https://github.com/ciur/papermerge/issues>`_ so I can fill in
-the gaps.
 
-1. Within extracted directory copy ``papermerge.conf.py.example`` to ``/etc/``::
+If I follow this document and still have troubles, I can open an
```

Comment:
Documentation is written from second person perspective - not first one.

Reply:
I would have to jump back in the history, but I think I read both from you. I choose first one, can be 2nd. A simple change.